ruby-server-example/README.md

```diff
@@ -0,0 +1,66 @@
+This is an Example Ruby repo
+
+## Install nix
+```
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+
+```
+
+```
+To get started using Nix, open a new shell or run `. /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh`
+```
+You can experiment if the installation was successful by runing `nix run "nixpkgs#hello"`
+## Install bsf cli 
+nix profile install {{URL}}
+
+
+## Initialize the projct
+Now is the time to run `bsf init` which will generate the `bsf.hcl` and `bsf.locl` files. It also generates a bsf folder with all nix files requred to build the project. 
+
+## Add files before building 
+`git add . ` <- Required by nix sandboxing.
+
+## Develop the porject
+This will get you in the development shell of your application
+`bsf develop`
+
+## Interact with the project
+You can checkout the ruby version
+`ruby -v`
+Try to install rails
+`gem install rails`
```

Comment:
`gem` should also be in bsf development shell

Reply:
Can you please elaborate this, I'm a bit confused with this!

Reply:
Where is the `gem` binary coming from? 
What is the output of `which gem` in development shell?

Reply:
So in `ruby-3.3.4` gem is installed by default